### PR TITLE
Minor improvements to the Python linter

### PR DESCRIPTION
**Pull request by mnvr**
Please see the individual commit messages for details.

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci please python lint

**Comment by CodaFi** [comment]
Going to manually squash and merge this

@swift-ci please smoke test

File: utils/python_lint.py [hunk 1]
```diff
@@ -23,9 +23,17 @@ def lint(arguments, verbose=True):
...
+The flake8 and flake8-import-order Python packages are required for linting,
+but these were not found on your system.
+
+You can install these using:
+
+    python -m pip install flake8
+    python -m pip install flake8-import-order
+
+For more help, see http://flake8.pycqa.org.""")
+        return flake8_result
```
Comment (CodaFi) [inline utils/python_lint.py]: You can keep the message, but it seems that our Linux bots lint gyb files and don’t have flake8 installed.  Please keep the return value as is.

Comment (mnvr) [reply utils/python_lint.py]: Thank you for catching this with a smoke test.

I've reverted the change, and added a sticky note.

With this reverted, there are no functional changes. So the commit message for the squash could read: "Improve Python linter documentation".

**Comment by CodaFi** [comment]
@swift-ci please smoke test

**Comment by CodaFi** [comment]
@swift-ci please smoke test

**Comment by CodaFi** [comment]
⛵️

**Pull request merged**